playbooks/common/openshift-master/config.yml

```diff
@@ -46,7 +46,7 @@
       openshift_hosted_metrics_duration: "{{ lookup('oo_option', 'openshift_hosted_metrics_duration') | default(7) }}"
     when: openshift_hosted_metrics_duration is not defined
   - set_fact:
-      openshift_hosted_metrics_resolution: "{{ lookup('oo_option', 'openshift_hosted_metrics_resolution') | default(10) }}"
+      openshift_hosted_metrics_resolution: "{{ lookup('oo_option', 'openshift_hosted_metrics_resolution') | default('10s', true) }}"
```

Comment:
I don't think these set_facts are necessary? The default should be coming from [openshift_facts](https://github.com/openshift/openshift-ansible/blob/00eba039c9312fbd04cc05a8a890ef48f2311769/roles/openshift_facts/library/openshift_facts.py#L1725-L1764).